components/pedigree/resources/src/main/resources/pedigree/saveLoadEngine.js

```diff
@@ -193,7 +193,8 @@ define([
                         // since we just loaded data from disk data in memory is equivalent to data on disk
                         editor.getUndoRedoManager().addSaveEvent();
                     } else {
-                        new TemplateSelector(true);
+                        document.fire("pedigree:load:finish");
+                        $('action-templates').click();
```

Comment:
It basically opens the dialog with tabs with some additional settings, a listener is attached to a button and its function has some lines of code, pedegree.js lines 189-193.